jsonpatcher/jsonpatcher.go

```diff
@@ -249,4 +261,34 @@ func (je patchEvent) Collection() string {
 	return je.CollectionName
 }
 
+type patchEventJson struct {
+	Timestamp      int64         `json:"timestamp"`
+	ID             string        `json:"_id"`
+	CollectionName string        `json:"collection_name"`
+	Patch          operationJson `json:"patch"`
+}
+
+type operationJson struct {
+	Type       string      `json:"type"`
+	InstanceID string      `json:"instance_id"`
+	JSONPatch  interface{} `json:"json_patch"`
+}
+
+func (je patchEvent) Marshal() ([]byte, error) {
```

Comment:
This uses some custom types that have `json` decorators

Reply:
Is this breaking change? Saying: changing the json format in the records?
Thinking about if older code can unmarshal correctly the new json tags? Sorry if confused!

Reply:
I created these new `Json` flavored structs (`patchEventJson` and `operationJson`) to avoid that problem. `Marshal` first unmarshals the patch to an empty interface, which is used to manually create a `patchEventJson` object, which is finally marshaled to JSON.